pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -1109,8 +1138,15 @@ func aceToRules(ctx *zedrouterContext, aclArgs types.AppNetworkACLArgs,
 		}
 	}
 	if foundDrop {
-		outActions = append(outActions, []string{"-j", "DROP"}...)
-		inActions = append(inActions, []string{"-j", "DROP"}...)
+		if aclArgs.NIType == types.NetworkInstanceTypeSwitch {
```

Comment:
For local NI we still want to drop packets between nbu1x1 and nbu1x2. Do we not need this drop rule for that?

Reply:
For me it was also quite surprising, but even if packets go between apps on the same bridge, they are still routed by EVE and therefore the IP rule with dummy output interface is applied. For example:
```
15:13:36.264140 nbu1x2 P   IP 10.11.12.3 > 10.11.12.2: ICMP echo request, id 369, seq 8, length 64
15:13:36.264140 bn2   In  IP 10.11.12.3 > 10.11.12.2: ICMP echo request, id 369, seq 8, length 64
15:13:36.271049 flow-mon-dummy Out IP 10.11.12.3 > 10.11.12.2: ICMP echo request, id 369, seq 8, length 64
```
We already rely on this for the default-drop-all ACE (i.e. there are no DROP rules for local NIs in either raw or filter table).

Reply:
Actually this is because applications are configured with this route (based on DHCP info they obtain):
`10.11.12.0/24 via 10.11.12.1 dev eth0`
However, just adding a link local route `ip route add 10.11.12.2 dev eth0`, makes packets go directly between apps, therefore skipping routing in EVE. And so there already is this vulnerability for the default-drop ACE tin the current master. I will work on a fix for both the implicit drop ACE as well as for explicit drop ACEs.

Reply:
OK can do that as a separate PR

Reply:
OK, created a separate story for this.